### DISCUSSION TITLE
为什么PCL还没有深浅色模式

### DISCUSSION THREAD

**Discussion by EyKettle** [Meloong-Git/PCL#2588] [提问] [answered]
现在绝大部分应用都有深色模式了，晚上打开PCL过于刺眼。

**Comment by iPodtouch6557** [comment, marked as answer]
#131 #1084 #1500 #2198
另外，解决晚上打开PCL很刺眼的最好办法是调一个纯黑色背景图片，~或者干脆别在半夜开PCL，也对眼睛不好~

**Reply by EyKettle** [reply]
我不喜欢用背景，任何软件都是默认的空白皮肤。我更希望应用自身有黑暗色系的界面。

**Reply by iPodtouch6557** [reply]
可是现在就是没有这个功能，而且龙猫也暂不考虑，能有啥办法, :(
或者也可以去留个言，表示迫切想要，可能能争取到一个暂缓处理……

**Comment by WForst-Breeze** [comment]
#131 如果您迫切需此功能，可以在这条 issue 下方发表您的观点理由和想法，龙猫会看到的。
